tapir/shifts/templates/shifts/shift_detail.html

```diff
@@ -56,122 +56,123 @@ <h5 class="d-flex justify-content-between align-items-center">
                 </span>
             {% endif %}
         </div>
-        <ul class="list-group list-group-flush">
-            {% if shift.description %}<li class="list-group-item">{{ shift.description }}</li>{% endif %}
-            {% if shift.cancelled %}
-                <li class="list-group-item bg-warning">
-                    {% translate 'This shift has been cancelled: ' %}
-                    {{ shift.cancelled_reason }}
-                </li>
-            {% endif %}
-            <li class="list-group-item">
-                <table class="{% tapir_table_classes %}"
-                       id="attendance_table"
-                       aria-label="{% translate 'List of slots for this shift' %}">
-                    <thead>
-                        <tr>
-                            <th>{% translate "Slot" %}-{% translate 'Number' %}</th>
-                            <th>{% translate 'Details' %}</th>
-                            <th>{% translate 'Registered user' %}</th>
-                            <th>{% translate 'Attendance' %}</th>
-                            <th>{% translate 'Actions' %}</th>
-                            <th>{% translate 'Do you meet the requirements?' %}</th>
-                            {% if perms.shifts.manage %}
-                                <th>{% translate 'Member-Office actions' %}</th>
-                                <th>{% translate 'Previous attendances' %}</th>
-                            {% endif %}
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {% for slot in slots %}
+        <div class="card-body d-flex flex-wrap">
+            <ul class="list-group list-group-flush d-flex flex-wrap">
+                {% if shift.description %}<li class="list-group-item">{{ shift.description }}</li>{% endif %}
+                {% if shift.cancelled %}
+                    <li class="list-group-item bg-warning">
+                        {% translate 'This shift has been cancelled: ' %}
+                        {{ shift.cancelled_reason }}
+                    </li>
+                {% endif %}
+                <li class="list-group-item">
```

Comment:
I think the `list-group` should be next to `card-body`, not in it. Or maybe replacing card-body.
Inside card body: 
![image](https://github.com/user-attachments/assets/3c83b094-a83d-4a9d-b778-27ef141e92d7)
Outside card body:
![image](https://github.com/user-attachments/assets/9e709cfa-818e-453c-be01-7160515c2c02)